Removed some things

cdedwards opened the pull request:
Removed some stuff not required/needed for a CNC or Laser.

cdedwards committed:
initital start

first commit. Bunch of unneeded stuff removed from the display for CNC/Lasers

cdedwards committed:
Create Filaments.vue

Added back Filaments.vue

Sindarius commented:
So some of this touches shared ui with DWC proper. The eventual goal I took with my approach was not to deviate the UI completely but to eventually make it where both UI would co-exist. You'll notice if you load the DWC-CNC and have it set to FFF mode the original UI is retained.

cdedwards commented:
I know. My lack of knowledge on vue programming or for that matter web programming is showing.. gimme some proper c code and I'll shine.  Someone somewhere might find this useful though

cdedwards committed:
Update CNCContainerPanel.vue

cdedwards committed:
Create CNCSettingsListItemsPanel.vue

cdedwards committed:
revamped SettingsListItemsPanel

Revamped to change dynamically when in FFF,CNC or laser mode excluding some unneeded features  in CNC/Laser modes

cdedwards commented:
Updated the pull. It's still not correct
